apps/src/pythonlab/constants.ts

```diff
@@ -74,3 +74,6 @@ export const STANDALONE_NEIGHBORHOOD_PROJECT: ProjectSources = {
 export const PYTHONLAB_EDITABLE_FILE_TYPES = ['py', 'csv', 'txt'];
 
 export const PYTHONLAB_SUPPORTED_FILE_TYPES = PYTHONLAB_EDITABLE_FILE_TYPES;
+
+export const PYTHONLAB_TUTOR_ADDITIONAL_CONTEXT =
+  'The student can view the documentation by clicking the book icon at the top of the workspace.';
```

Comment:
the documentation button is about to move fyi: https://github.com/code-dot-org/code-dot-org/pull/68993